modules/enabling-team-sync-oidc.adoc

```diff
@@ -0,0 +1,210 @@
+:_content-type: PROCEDURE
+[id="oidc-team-sync"]
+= Team synchronization for {productname} OIDC deployments
+
+Administrators can leverage an OpenID Connect (OIDC) identity provider that supports group or team synchronization to apply repository permissions to sets of users in {productname}. This allows administrators to avoid having to manually create and sync group definitions between {productname} and the OIDC group.
+
+:_content-type: PROCEDURE
+[id="enabling-oidc-team-sync"]
+== Enabling synchronization for {productname} OIDC deployments
+
+Use the following procedure to enable team synchronization when you {productname} deployment uses an OIDC authenticator.
+
+[IMPORTANT]
+====
+The following procedure does not use a specific OIDC provider. Instead, it provides a general outline of how best to approach team synchronization between an OIDC provider and {productname}. Any OIDC provider can be used to enable team synchronization, however, setup might vary depending on your provider.
+====
+
+.Procedure
+
+. Update your `config.yaml` file with the following information:
++
+[source,yaml]
+----
+AUTHENTICATION_TYPE: OIDC
+# ...
+OIDC_LOGIN_CONFIG:
+  CLIENT_ID: <1>
+  CLIENT_SECRET: <2>
+  OIDC_SERVER: <3>
+  SERVICE_NAME: <4>
+  PREFERRED_GROUP_CLAIM_NAME: <5>
+  LOGIN_SCOPES: [ '', '' ] <6>
+# ...
+FEATURE_TEAM_SYNCING: true <7>
+FEATURE_NONSUPERUSER_TEAM_SYNCING_SETUP: true <8>
+FEATURE_V2_UI: true
+# ...
+----
+<1> Required. The registered OIDC client ID for this {productname} instance.
+<2> Required. The registered OIDC client secret for this {productname} instance.
+<3> Required. The address of the OIDC server that is being used for authentication.
```

Comment:
That url should be such that GET `<OIDC_SERVER>/.well-known/openid-configuration` should return the providers configuration as mentioned [here](https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfig).